releases/4.0.0-beta.1.md

```diff
@@ -5,13 +5,13 @@ category: Releases
 
 # TiDB 4.0.0-beta.1 Release Notes
```

Comment:
I see that you update the title to `TiDB 4.0.0 Beta.1 Release Notes` in https://github.com/pingcap/docs/pull/2262. Keep them consistent? And please also update `title` in metadata.
```suggestion
# TiDB 4.0.0 Beta.1 Release Notes
```